docs/src/main/asciidoc/_developer-guide/Jakarta_XML_Web_Services/Apache_CXF_integration.adoc

```diff
@@ -427,8 +427,8 @@ Schema validation of exchanged messages can also be enabled in
 `jboss-webservices.xml`. Further details available
 <<Jakarta_XML_Web_Services_User_Guide,here>>.
 
-[[interceptors]]
-==== Interceptors
+[[Jakarta_Interceptors]]
+==== Jakarta Interceptors
```

Comment:
I believe this should not be changed, as the interceptors discussed here are Apache CXF interceptors, not JavaEE/JakartaEE interceptors. @jimma , please confirm.

Reply:
Here is CXF interceptor and we don't need to change it. @asoldano @RanabirChakraborty @yersan